### PR TITLE
add tutorial doc in website repo about how to do migration rollback

### PR DESCRIPTION
**Pull request by wulemao**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

add tutorial doc in website repo about how to do migration rollback

**Which issue(s) this PR fixes**:

Fixes part of https://github.com/karmada-io/karmada/issues/5380

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by samzong** [comment]
If in WIP, maybe you can change the PR status to Draft.

File: docs/tutorials/resource-migration.md
Comment (samzong) [inline docs/tutorials/resource-migration.md:1]: Whether using promote is the **correct** way.

follow this document, I think the behavior of migration in the tutorial is a bit like "overwriting"

why does not use 'karmadactl promote deployment'?

Comment (samzong) [reply docs/tutorials/resource-migration.md:1]: like this. https://karmada.io/zh/docs/administrator/migration/promote-legacy-workload

Comment (wulemao) [reply docs/tutorials/resource-migration.md:1]: `karmadactl promote deployment` command is limited, that command can only migrate a specified resource.

Comment (wulemao) [reply docs/tutorials/resource-migration.md:1]: In other words, this is an exposed API, while karmadactl is only a simple encapsulation of this API, with limited capability and not as widely used as the original API.

Comment (samzong) [reply docs/tutorials/resource-migration.md:1]: okay, Need @RainbowMango  help evaluating and whether the promote docs needs to be updated

Comment (RainbowMango) [reply docs/tutorials/resource-migration.md:1]: > Whether using promote is the correct way.

My two cents.
There is nothing about correct or not, just fit or not. 
Karmada enables different ways to migrate legacy applications, people decide to use the one that fits them well.

File: docs/tutorials/resource-migration.md [hunk 1]
```diff
@@ -9,8 +9,8 @@ you want to migrate the existing resource to Karmada and then achieve multi-clus
 
 So, this section will guide you to cover:
 
-- Migrate all the existing resource from original cluster to Karmada based on resource granularity.
-- Apply higher priority `PropagationPolicy` to meet more propagate demands based on application granularity.
+- Migrating resources that are already deployed in the original single cluster to Karmada.
+- Rolling back the migration, removing the migrated resources from Karmada while preserving the resources in the member clusters.
```
Comment (RainbowMango) [inline docs/tutorials/resource-migration.md:13]:
```suggestion
- Migrate application from Kubernetes to Karmada.
- Rollback application migration.
```

File: docs/tutorials/resource-migration.md [hunk 2]
```diff
@@ -138,98 +110,92 @@ spec:
     clusterAffinity:
       clusterNames:
       - member1
-  priority: 0
+      - member2
```
Comment (RainbowMango) [inline docs/tutorials/resource-migration.md:113]: In the migration process, it is not recommended to change the application topology. 

I don't think `member2` should be present here.
And the following content should be updated accordingly.

File: docs/tutorials/resource-migration.md [hunk 3]
```diff
@@ -138,98 +110,92 @@ spec:
...
-You shall see the Deployments in Karmada are all ready and the `aggregatedStatus` of `ResourceBinding` is applied, 
-which means the existing resources in `member1` cluster has been taken over by Karmada. 
+You will see that all Deployments in Karmada are ready, among them:
 
-So far, you have finished the migration, isn't it so easy?
+* The `nginx-deploy` in the member1 cluster successfully taken over by Karmada:
+  * `AGE=16m` indicates the resource was taken over and not recreated, so the corresponding pods do not restart.
+  * `ADOPTION=Y` indicates that the resource is now managed by Karmada.
+* The member2 cluster, which previously had no `nginx-deploy`, has successfully received the deployment in replication mode, 
+  with `ADOPTION=Y` indicating it is now managed by Karmada.
```
Comment (RainbowMango) [inline docs/tutorials/resource-migration.md:161]: Deploying to members is not the intention of this documentation. 
This doc should focus on demonstrating migration.

File: docs/tutorials/resource-migration.md [hunk 4]
```diff
@@ -138,98 +110,92 @@ spec:
...
 
-### Apply higher priority PropagationPolicy
+* You have now completed the migration. Isn't it simple?
```
Comment (RainbowMango) [inline docs/tutorials/resource-migration.md:163]:
```suggestion

```
remote this line.

File: docs/tutorials/resource-migration.md [hunk 5]
```diff
@@ -138,98 +110,92 @@ spec:
...
 
-#### Step 5: Write the code
+### Rolling Back Migrated Resources
```
Comment (RainbowMango) [inline docs/tutorials/resource-migration.md:165]:
```suggestion
### Rollback Migration
```

**Review by RainbowMango — COMMENTED** [review]
This review only focuses on the new content.

File: docs/tutorials/resource-migration.md (hunk 4 above)
Comment (RainbowMango) [reply docs/tutorials/resource-migration.md:163]: Or remove `Isn't it simple?`

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**